examples/Remarketing/AddCombinedRuleUserList.php

```diff
@@ -0,0 +1,203 @@
+                    PHP_EOL
+                );
+            }
+            exit(1);
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+            exit(1);
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     */
+    public static function runExample(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId
+    ) {
+        // Creates a rule targeting any user that visited a url that equals
+        // http://example.com/example1'.
+        $userVisitedSite1Rule = new UserListRuleItemInfo([
+            // Uses a built-in parameter to create a domain URL rule.
+            'name' => new StringValue(['value' => self::URL_STRING]),
+            'string_rule_item' => new UserListStringRuleItemInfo([
+                'operator' => UserListStringRuleItemOperator::EQUALS,
+                'value' => new StringValue(['value' => 'http://example.com/example1'])
+            ])
+        ]);
+
+        // Creates a UserListRuleInfo object containing the first rule.
+        $userVisitedSite1RuleInfo = new UserListRuleInfo([
+            'rule_item_groups' => [
+                new UserListRuleItemGroupInfo([
+                    'rule_items' => [$userVisitedSite1Rule]
+                ])
+            ]
+        ]);
+
+        // Creates a rule targeting any user that visited a url that equals
+        // http://example.com/example2'.
+        $userVisitedSite2Rule = new UserListRuleItemInfo([
+            // Uses a built-in parameter to create a domain URL rule.
+            'name' => new StringValue(['value' => self::URL_STRING]),
+            'string_rule_item' => new UserListStringRuleItemInfo([
+                'operator' => UserListStringRuleItemOperator::EQUALS,
+                'value' => new StringValue(['value' => 'http://example.com/example2'])
+            ])
+        ]);
+
+        // Creates a UserListRuleInfo object containing the second rule.
+        $userVisitedSite2RuleInfo = new UserListRuleInfo([
+            'rule_item_groups' => [
+                new UserListRuleItemGroupInfo([
+                    'rule_items' => [$userVisitedSite2Rule]
+                ])
+            ]
+        ]);
+
+        // Creates the user list where "Visitors of a page who did visit another page". To create a
+        // user list where "Visitors of a page who did not visit another page", change the
+        // UserListCombinedRuleOperator from PBAND to AND_NOT.
+        $combinedRuleUserListInfo = new CombinedRuleUserListInfo([
+            'left_operand' => $userVisitedSite1RuleInfo,
+            'right_operand' => $userVisitedSite2RuleInfo,
+            'rule_operator' => UserListCombinedRuleOperator::PBAND
```

Comment:
Assuming AND -> PBAND is a PHP quirk?

Reply:
I assumed AND is a language keyword so they had to change for **P**rotocol**B**uffers**AND**